lib/fastlane/actions/slack.rb

```diff
@@ -63,12 +65,30 @@ def self.available_options
                                        env_name: "FL_SLACK_CHANNEL",
                                        description: "#channel or @username",
                                        optional: true),
+          FastlaneCore::ConfigItem.new(key: :override_slack_username_and_icon,
```

Comment:
i don't think we need this option, if the two other options are supplied they can override the default values that i mentioned above